cmd/reva/download.go

```diff
@@ -38,7 +37,7 @@ import (
 	"github.com/studio-b12/gowebdav"
 )
 
-func downloadCommand() *command {
+func downloadCommand(httpClient *http.Client) *command {
```

Comment:
`client` is a global variable, you don't need to pass it as a function argument

Reply:
Ah, sorry. Fixed it.